### PR TITLE
Fix pymedusa/Medusa/issues/1387

### PR DESCRIPTION
**Pull request by duramato**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read [contribution guide](https://github.com/PyMedusa/SickRage/blob/master/.github/CONTRIBUTING.md)

@medariox

### CODE REVIEW

**Comment by medariox** [comment]
Why append and Regex? I'd go for replace

**Comment by duramato** [comment]
My tought here we need "Season 2" and "S02" becouse the 2 are avaiable on the site, so we need to search for the 2 formats becouse we don't know when there will be one or the other.

**Comment by fernandog** [comment]
@medariox has to be re.sub : 

{u'Season': [u'Arrow S04']}

**Comment by fernandog** [comment]
i have a better solution. testing now

File: medusa/providers/torrent/html/morethantv.py [hunk 1]
```diff
@@ -88,6 +88,9 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
         for mode in search_strings:
             logger.log('Search mode: {0}'.format(mode), logger.DEBUG)
 
+            if mode == 'Season':
+                search_strings[mode].append(re.sub(r'(.*)S0?', r'\1Season ', search_strings[mode][0]))
```
Comment (fernandog) [inline medusa/providers/torrent/html/morethantv.py]: problem of this is that if there is scene exceptions it will use only the first one.

**Comment by fernandog** [comment]
@medariox are you going to try parse season in this PR? 

@duramato can you test this?

**Comment by fernandog** [comment]
rebased

**Pull request merged**